app/src/main/java/org/mozilla/fenix/settings/TabsSettingsFragment.kt

```diff
@@ -40,9 +40,15 @@ class TabsSettingsFragment : PreferenceFragmentCompat() {
     private lateinit var inactiveTabsCategory: PreferenceCategory
     private lateinit var inactiveTabs: SwitchPreference
     private lateinit var searchTermTabGroups: SwitchPreference
+    /**
+     * Turning off inactive tabs survey
+
     private val shouldShowInactiveTabsTurnOffSurvey
         get() = requireContext().settings().isTelemetryEnabled &&
-            requireContext().settings().shouldShowInactiveTabsTurnOffSurvey
+                requireContext().settings().shouldShowInactiveTabsTurnOffSurvey
+    **/
+    private val shouldShowInactiveTabsTurnOffSurvey = false
```

Comment:
I was expecting this change in Settings.kt actually. You can set the default value of the SharedPref to false. Otherwise, now we have an unused SharedPrefs for `shouldShowInactiveTabsTurnOffSurvey`

Reply:
Fixed